### PR TITLE
Refactor about page v2

### PR DESCRIPTION
**Pull request by railsstudent**
Externalize details of contributors in private/about.json and read the data from it when rendering the images and popups in about.html.

When new contributors wants to add themselves to about page, they can append JSON object of personal details in code-contributors array.   

The html code in title attribute in original design is errone hone with all these angle brackets

### CODE REVIEW

**Comment by railsstudent** [comment]
@lpatmo  Yes, you can move my image to core contributor session.
My slack handler name is @Connie
twitter is con_leung
github is railsstudent

Thanks.

File: private/about.json [hunk 1]
```diff
@@ -0,0 +1,235 @@
+{
```
Comment (distalx) [inline private/about.json]: We should move this file under `imports/data` directory, and break it down to two files, 
The scope of the private directory is limited to server side only.

very roughly : 

`imports/data/flags.json` and `imports/data/contributors.json`
```
[
  {
     name: 'xyz',
     avatar: 'http://link or gravater',
     intro: 'lorem ipsum',
     socials:[
         {website: 'URL'},
         {twitter: '@handel'},
     ....
     ],
     type: 'code'
  },
   ....
]
```

on about page you can load this `json` files with their relative path
```
imports contributors from 'imports/data/contributors.json'
```
with the help of the helper function contributors can be passed to the template 
```
`contributors : function() { return contributors }`
```
and then we can iterate over it using 
`{{#each contributors}} {{/each}}` block.

To display contributors into their respective section we can use `equals` template helper  `{{#if equals type 'code'}}`

Comment (railsstudent) [reply private/about.json]: I do not see imports/data directory in the project. Should I create it? thanks.

Comment (distalx) [reply private/about.json]: Yes, `imports` directory is a part of a new directory structure suggested in meteor guide, our migration to the new directory structure is long over due.

Comment (railsstudent) [reply private/about.json]: Noted. Will get this done before Sunday.

Comment (railsstudent) [reply private/about.json]: @distalx  Please kindly review

**Comment by distalx** [comment]
@railsstudent , Thanks for the pull request.

**Pull request merged**